homelab/hosts/mac-mini/silverbullet/scripts/bootstrap.sh

```
@@ -0,0 +1,150 @@
+
+env_value() {
+  local key="$1"
+  grep -F "${key}=" "$ENV_FILE" | tail -1 | cut -d= -f2- \
+    | sed -e 's/^"//' -e 's/"$//' -e "s/^'//" -e "s/'$//"
+}
+
+SB_USER="$(env_value SB_USER)"
+if [[ -z "$SB_USER" || "$SB_USER" == "CHANGEME" ]]; then
+  echo "SB_USER is not set or still CHANGEME in $ENV_FILE" >&2
+  echo "Set it to user:password, e.g.:" >&2
+  printf '  printf "SB_USER=%%s:%%s\\n" "$(whoami)" "$(openssl rand -base64 18)" >> %s\n' "$ENV_FILE" >&2
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)